services/QuillLMS/app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class ApplicationMailer < ActionMailer::Base
```

Comment:
We have a linting rule not to allow new classes to inherit from `Action____::Base`, so we need to create intermediate `Application____` models to inherit from.  This is the first `Mailer` that we've written since that rule got put into place.